### PR TITLE
add total participants metrics

### PR DESCRIPTION
**Pull request by mclpfr**
Hello,
I need "total_participants" metrics to make some graphs
thx

### CODE REVIEW

**Comment by xshadow** [comment]
The tests for this PR fail, you have to add tests in main_test.go

**Comment by mclpfr** [comment]
Hello,
Thank you for your feedback, I updated it.
Best regards

File: main_test.go [hunk 1]
```diff
@@ -68,6 +68,9 @@ jitsi_p2p_conferences 1
 # HELP jitsi_participants The current number of participants.
 # TYPE jitsi_participants gauge
 jitsi_participants 4
+# HELP jitsi_total_participants Total participants since running.
+# TYPE jitsi_total_participants gauge
+jitsi_total_participants 20
```
Comment (0x46616c6b) [inline main_test.go:73]:
```suggestion
jitsi_total_participants 18
```

File: main_test.go [hunk 2]
```diff
@@ -279,6 +282,9 @@ jitsi_p2p_conferences 0
 # HELP jitsi_participants The current number of participants.
 # TYPE jitsi_participants gauge
 jitsi_participants 0
+# HELP jitsi_total_participants Total participants since running.
+# TYPE jitsi_total_participants gauge
+jitsi_total_participants 20
```
Comment (0x46616c6b) [inline main_test.go:287]:
```suggestion
jitsi_total_participants 2
```

**Review by 0x46616c6b — CHANGES_REQUESTED** [review]
I added two suggestions to fix the test.

File: main_test.go (hunk 1 above)
Comment (mclpfr) [reply main_test.go]: Thank you, it's fixed

File: main_test.go (hunk 2 above)
Comment (mclpfr) [reply main_test.go]: Thank you, it's fixed

**Pull request merged**